indico/modules/events/tracks/schemas.py

```diff
@@ -17,5 +19,10 @@ class Meta:
         fields = ('id', 'title', 'code', 'description')
 
 
+class TrackPermissionsSchema(mm.ModelSchema):
```

Comment:
do you have to inherit from `mm.ModelSchema` here as well? (similar to my comment above)